addons/project_todo/static/tests/todo_form_view.js

```diff
@@ -0,0 +1,102 @@
+/* @odoo-module */
+
+import { startServer } from "@bus/../tests/helpers/mock_python_environment";
+import { addModelNamesToFetch } from "@bus/../tests/helpers/model_definitions_helpers";
+
+import { start } from "@mail/../tests/helpers/test_utils";
+
+import { click, getFixture } from "@web/../tests/helpers/utils";
+import { contains } from "@web/../tests/utils";
+import { setupViewRegistries } from "@web/../tests/views/helpers";
+
+import { patchUserWithCleanup } from "@web/../tests/helpers/mock_services";
+
+addModelNamesToFetch(["project.task"]);
+
+let serverData;
+let target;
+
+QUnit.module("todoFormView Tests", (hooks) => {
+    hooks.beforeEach(async () => {
+        const pyEnv = await startServer();
+        pyEnv["project.task"].create([{ name: "Todo 1", state: "01_in_progress" }]);
+        serverData = {
+            views: {
+                "project.task,false,form": `
+                    <form js_class="todo_form">
+                        <field name="name"/>
+                        <field name="state"/>
+                    </form>`,
+                "project.task,false,list": `
+                    <tree>
+                        <field name="name"/>
+                        <field name="state"/>
+                    </tree>`,
+            },
+        };
+        target = getFixture();
+        setupViewRegistries();
+    });
+
+    QUnit.test("Check that project_task_action_convert_todo_to_task appears in the menu actions if the user does belong to the group_project_user group", async function (assert) {
+        assert.expect(1);
+        const { openView } = await start({
+            serverData,
+        });
+        await openView({
+            res_model: "project.task",
+            views: [
+                [false, "list"],
+                [false, "form"],
+            ],
+        });
+        patchUserWithCleanup({ hasGroup: () => Promise.resolve(true) }); // For the user to have the group_project_user group and access to project_task_action_convert_todo_to_task
```

Comment:
Should we not just give the user the ``group_project_user`` group like so?
```suggestion
        patchUserWithCleanup({ hasGroup: (group) => group === 'group_project_user'}); 
```
I suppose this applies to all instances of such testing :thinking:

Reply:
Yes it is more accurate indeed! :D